### PR TITLE
Hall of Fame

### PR DESCRIPTION
**Pull request by huayunh**
#### Changes proposed in this Pull Request:
- Guys please add your own description & whatever profiles by editing 
```doc-it/src/__configuration__/contributors/currentMaintainers.ts```
- Update your profile image by uploading your images to 
```doc-it/src/app/assets/credits```
- I experimentally added a section for "A Brief History of PX Blue" at `doc-it/src/docs/community/our-team.mdx`, but I don't know what to write there. Someone _a.k.a. Joe_ will have to expand that section or just remove it. 
- The contributor list might not be complete. Feel free to update 
```doc-it/src/__configuration__/contributors/contributors.ts``` as well
- Question: do you think Shivani and Kraig will sue us for stealing their GitHub picture?

#### Screenshots:

Two pages will be affected: 

`/community/our-team`
image

and the landing page, where we show off how many contributors we have for this pxblue project

image

### CODE REVIEW

**Comment by joebochill** [comment]
> * Question: do you think Shivani and Kraig will sue us for stealing their GitHub picture?

I don't think so. But it might be worth reaching out to the people we're planning to list here to make sure they are okay with us using their name on the site. I don't suspect anybody will have a problem, but you never know.

**Comment by joebochill** [comment]
As a possible suggestion, it might be cool to do something like this instead of the regular button:
image
image

**Comment by joebochill** [comment]
I think this looks really awesome. Nice job

File: src/__configuration__/contributors/currentMaintainers.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import ThomasDailey from '../../app/assets/credits/thomas-dailey.jpeg';
+import KyleHorchen from '../../app/assets/credits/kyle-horchen.jpg';
+import { Contributor } from '../../__types__';
+
+export const currentMaintainers: Contributor[] = [
+    {
+        name: 'Huayun Huang',
+        description: 'UX Designer & Developer',
+        image: HuayunHuang,
+        info: `Huayun is an interesting half blood between a UX-driven designer and a detailed-oriented developer. She likes spending her spare time on 2D visual arts. Be prepared! She has no mercy in your review requests.`,
```
[the remark this answers is not in the document]
Comment (huayunh) [reply src/__configuration__/contributors/currentMaintainers.ts:14]: Good point, I will add that in :P 

"Thus we offer you pxblue care 24/5"

**Comment by emclaug2** [comment]
Hey here's my description:

Evan is a local to the Pittsburgh area and has been doing front-end web development since graduating from the University of Rochester.   He is an Angular specialist and insists that it's better than whatever else you're using.   Evan's favorite coworker is his dog, Juniper, who helps debug his code daily.

**Comment by daileytj** [comment]
Heres me description:

Basement dwelling Pittsburgh native and sworn enemy of Internet Explorer, Tom is a self taught font-end web developer who will stop at nothing to prove himself. When he’s not busy programming you can find him playing guitar or enjoying the great outdoors with his wife and daughter.

**Comment by jeffvg** [comment]
Here is my description

Jeff is a perfectionist. With over 14 years of experience in software testing and as a Software Engineer in Test, Jeff continues to be passionate about quality and the journey that it takes to produce the best results. When Jeff is not testing, he can be found doing home remodeling with his wife or turning wrenches.

**Comment by huayunh** [comment]
Per discussion on Teams with Joe, since we are not really hiring, I'd drop that "Join our team" thing for now; the remaining "become a contributor" avatar list thing Joe proposed earlier is not as encouraging as the existing design, sooo... I changed the API, but I am not including the avatar list design for this round.

**Comment by huayunh** [comment]
And as always, waiting for the master branch to `yarn indexer`

**Comment by huayunh** [comment]
> But it might be worth reaching out to the people we're planning to list here to make sure they are okay with us using their name on the site. I don't suspect anybody will have a problem, but you never know.

Time for a premium LinkedIn account!

**Review by joebochill — APPROVED** [review]
I approve...check out my other PR with some slight wording changes if you dare.

**Comment by huayunh** [comment]
> check out my other PR with some slight wording changes if you dare.

You are trying to maintain your bio's correctness already 🤣

**Pull request merged**